fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -483,7 +483,7 @@ proxy_options() {
 
   local noProxy="${no_proxy:-${NO_PROXY:-}}"
   if [ -n "$noProxy" ] ; then
-    ret="$ret -Dhttp.nonProxyHosts=\"$(echo "|$noProxy" | sed -e 's/,[[:space:]]*/|/g' | sed -e 's/|\./|\*\./g' | cut -c 2-)\""
+    ret="$ret -Dhttp.nonProxyHosts=$(echo "|$noProxy" | sed -e 's/,[[:space:]]*/|/g' | sed -e 's/|\./|\*\./g' | cut -c 2-)"
```

Comment:
Not sure whether there can be still spaces in a noProxy spec, so still keeping the quotes would be safe in any case. Let's find out why the test is failing when quotes are provided.

Reply:
Not sure what the actual fix is. Is it just removing the quotes ?

Reply:
yes this is the actual fix